### PR TITLE
Update javadoc of capabilities to follow latest Appium documents

### PR DESCRIPTION
**Pull request by takeya0x86**
## Change list

Update javadoc of capabilities to follow latest Appium documents.
https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/caps.md
and
https://github.com/appium/appium-xcuitest-driver/blob/master/README.md#desired-capabilities
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] No changes in production code.
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted java code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: src/main/java/io/appium/java_client/remote/IOSMobileCapabilityType.java [hunk 1]
```diff
@@ -69,6 +74,7 @@
      * Dismiss all iOS alerts automatically if they pop up.
      * This includes privacy access permission alerts (e.g.,
      * location, contacts, photos). Default is false.
+     * Does not work on XCUITest-based tests.
```
Comment (mykola-mokhnach) [inline src/main/java/io/appium/java_client/remote/IOSMobileCapabilityType.java]: it does starting from Appium 1.13.0

File: src/main/java/io/appium/java_client/remote/IOSMobileCapabilityType.java [hunk 2]
```diff
@@ -175,8 +181,10 @@
     String TAP_WITH_SHORT_PRESS_DURATION = "tapWithShortPressDuration";
 
     /**
-     * The capability to direct Appium to set the simulator scale.
-     * The  XCUITest specific capability.
+     * Simulator scale factor.
```
Comment (mykola-mokhnach) [inline src/main/java/io/appium/java_client/remote/IOSMobileCapabilityType.java]: This only works below Xcode9

**Comment by mykola-mokhnach** [comment]
Thanks @takeya0x86 . Please address linter issues from https://travis-ci.org/appium/java-client/builds/570742270?utm_source=github_status&utm_medium=notification

File: src/main/java/io/appium/java_client/remote/IOSMobileCapabilityType.java (hunk 1 above)
Comment (takeya0x86) [reply src/main/java/io/appium/java_client/remote/IOSMobileCapabilityType.java]: Done. delete this line.

File: src/main/java/io/appium/java_client/remote/IOSMobileCapabilityType.java (hunk 2 above)
Comment (takeya0x86) [reply src/main/java/io/appium/java_client/remote/IOSMobileCapabilityType.java]: Added comment below this line.

**Comment by takeya0x86** [comment]
@mykola-mokhnach Thank you for reviewing. I fixed these. And I also updated appium capability documents.

File: src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java [hunk 3]
```diff
@@ -19,29 +19,39 @@
...
+     * By default the value of this capability is the same as for {@code appActivity}.
+     * You must set it to the very first focused application activity name in case it is different
+     * from the one which is set as {@code appActivity} if your capability has {@code appActivity}
+     * and {@code appPackage}.
      */
     String APP_WAIT_ACTIVITY = "appWaitActivity";
 
     /**
      * Java package of the Android app you want to wait for.
+     * By default the value of this capability is the same as for {@code appActivity}
```
Comment (mykola-mokhnach) [inline src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java]: We can also mention, that this cap can include wildcards (`*`)

Comment (takeya0x86) [reply src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java]: done.

**Pull request merged**